composer.json

```diff
@@ -11,12 +16,12 @@
         }
     ],
     "require": {
-        "php": ">=5.6",
-        "symfony/finder": "^2.7|^3.0|^4.0|^5.0"
+        "php": ">=7.1",
```

Comment:
If we drop php 5.6 and 7.0 support, shouldn't 5.6 and 7.0 be removed from `.travis.yml` too?

Reply:
I agree.
And more I think Travis-CI should be abandoned in favor of GH Actions.

Reply:
yes, i forgot to remove it. will work on it